apps/tutorial_servo_control_can/main.cpp

```diff
@@ -0,0 +1,19 @@
+#include "CANMsg.h"
+#include "TutorialServo.h"
+#include "mbed.h"
+
+CAN can(D3, D4);
+CANMsg msg;
+TutorialServo servo(PA_1, 180.0);
+
+int main() {
+  uint in;
+  can.frequency(500000);
+
+  while (true) {
+    if (can.read(msg)) {
+      msg >> in;
+    }
+    servo.setPositionInDegrees(180 * in / 100);
```

Comment:
instead of uint, it would make more sense for `in` to be a float and put directly into `setPositionInDegrees` but this is fine.